### ISSUE TITLE
[FEATURE] Docker-Compose script with resources

### ISSUE THREAD

**Issue by alexeyegorov** [cluster-apps-on-docker/spark-standalone-cluster-on-docker#40]
### Description

As I was trying out the 3.0.0 spark version and wanted to initialize workers with 2G or more, it failed every time. After looking up the [docker-compose wiki](https://docs.docker.com/compose/compose-file/), I've found that it is restricted by 2G as default and can be set in the docker-compose file. I would suggest to add these couple of lines for better understanding and setup of the local docker Spark cluster.

Here an example:

```
    deploy:
      resources:
        limits:
          cpus: '4'
          memory: 8G
```

**Comment by andre-marcos-perez** [comment]
Hi @alexeyegorov thanks again for your request, I have implemented your suggestions on #48

**Comment by andre-marcos-perez** [comment]
Hi @alexeyegorov, I actually remove it from 1.2.1 on #49 since it seems to be related to only to Docker Swarm, checkout the warning I got:

`WARNING: Some services (spark-worker-1, spark-worker-2) use the 'deploy' key, which will be ignored. Compose does not support 'deploy' configuration - use `docker stack deploy` to deploy to a swarm.`
